In [24]:
import pandas as pd

df = pd.read_csv(r'C:\Users\franc\Desktop\TESI\training_info\sensors_data.csv')
df


sensor0     sensor1     sensor2      target
0     869.612896  300.926005  298.750757  386.034598
1     896.568403  628.482642  742.601636  203.649900
2     863.767949  384.343307  405.575189  296.193265
3     858.161535  362.808724  380.003976  309.323387
4     853.102165  345.794242  862.369575  199.999204
...          ...         ...         ...         ...
5660  847.218363  300.111761  847.809820  200.000000
5661  844.130013  299.045144  842.396319  200.000005
5662  844.781470  297.608429  846.385450  199.999996
5663  845.091945  301.598523  843.119725  200.000008
5664  847.356684  297.486865  843.212972  200.000071

[5665 rows x 4 columns]

In [25]:
df_concat = pd.read_csv(r"C:\Users\franc\Desktop\TESI\SML_thesis_line_follower_robot\tmp\e-puck\data\data\sensors_data\sensors_data_drift.csv")
df_concat


sensor0     sensor1      sensor2      target
0     1687.583667  605.443081  1688.713407  200.000000
1     1687.248281  599.647811  1689.061903  199.999999
2     1689.030131  593.483542  1684.560693  200.000011
3     1687.137059  600.500179  1690.548340  199.999995
4     1685.247612  600.791106  1690.769440  199.999979
...           ...         ...          ...         ...
1995  1681.872385  601.535772  1692.807119  199.999837
1996  1685.515496  600.756620  1694.839496  199.999899
1997  1682.552917  599.439086  1690.293790  199.999942
1998  1688.899768  602.654912  1693.858920  199.999985
1999  1629.696813  597.453667  1693.727517  199.967185

[2000 rows x 4 columns]

# the model is trained on 5665 undrifted data and on 2000 drifted data

In [26]:
df = pd.concat([df, df_concat], ignore_index=True)
df

sensor0     sensor1      sensor2      target
0      869.612896  300.926005   298.750757  386.034598
1      896.568403  628.482642   742.601636  203.649900
2      863.767949  384.343307   405.575189  296.193265
3      858.161535  362.808724   380.003976  309.323387
4      853.102165  345.794242   862.369575  199.999204
...           ...         ...          ...         ...
7660  1681.872385  601.535772  1692.807119  199.999837
7661  1685.515496  600.756620  1694.839496  199.999899
7662  1682.552917  599.439086  1690.293790  199.999942
7663  1688.899768  602.654912  1693.858920  199.999985
7664  1629.696813  597.453667  1693.727517  199.967185

[7665 rows x 4 columns]